manager/container.go

```diff
@@ -226,12 +232,24 @@ func (c *containerData) housekeeping() {
 			}
 		}
 
-		// Schedule the next housekeeping. Sleep until that time.
+		// Run custom collectors.
+		nextCollectionTime, err := c.collectorManager.Collect()
```

Comment:
I was more thinking that it'd never be nil although there may be no collectors. I want to re-implement stats as a collector so there should be at least one collector.